tensorflow/contrib/go/doc.go

```diff
@@ -0,0 +1,9 @@
+/*
+Package tensorflow provides a high level Go API for TensorFlow, this package
+provides the necessary tools to create and manipulate Tensors, Variables,
+Constants and build, load and run Graphs.
+
+TensorFlow Go API allows you to load Graphs previously generated from Python,
```

Comment:
Ups, I forgot this when I was working on the MarkDowns, removed this two lines :)